internal/install/install.go

```diff
@@ -184,9 +187,14 @@ func writeKubernetesDeployerResources(elasticPackagePath *locations.LocationMana
 		return errors.Wrap(err, "can't read application configuration")
 	}
 
+	elasticAgentManagedYaml, err := DownloadFile("elastic-agent-managed-kubernetes.yaml", elasticAgentManagedYamlUrl)
+	if err != nil {
+		return errors.Wrap(err, "downloading elastic-agent-managed-kubernetes.yaml failed")
```

Comment:
I would replace the filename with something abstract in case the filename changes.